genai-perf/docs/goodput_tutorial.md

```diff
@@ -0,0 +1,134 @@
+<!--
```

Comment:
Please name this file consistently with the other tutorial docs. E.g. "goodput.md"

Reply:
Please see updated tutorial in a separate PR

Reply:
Perfect, nice job!